spec.md

```diff
@@ -532,23 +532,51 @@ Credentials are a set of information used by an application or a user to utilize
+services MUST include a `syslog_drain_url`. The platform MUST use the
+`syslog_drain_url` value when sending logs to the service.
 
-The `requires` field in the [Catalog](#catalog-management) endpoint enables a platform marketplace to validate a response for create binding that includes a `syslog_drain_url`. Platform marketplaces can consider a broker's response invalid if it includes a `syslog_drain_url` and `"requires":["syslog_drain"]` is not present in the [Catalog](#catalog-management) endpoint.
+Brokers MUST NOT include a `syslog_drain_url` in a create binding response
+if the associated [Catalog](#catalog-management) entry for the service
+did not include a `"requires":["syslog_drain"]` property.
 
 #### Route Services
 
-There are a class of service offerings that intermediate requests to applications, performing functions such as rate limiting or authorization. To configure a service instance with behavior specific to an application's routable address, a broker client MAY send the address along with the request to create a binding using `"bind_resource":{"route":"some-address.com"}`.
+There are a class of service offerings that intermediate requests to
+applications, performing functions such as rate limiting or authorization.
+If the platform and service both support this feature then the following
+applies:
 
-Some platforms MAY support proxying of application requests to service instances. In this case the platform needs to know where to send application requests; to facilitate this, the broker MUST return a `route_service_url` in the response for a request to create a binding. Not all services of this type expect to receive requests proxied by the platform; some services will have been configured out-of-band to intermediate requests to applications. In this case, the broker will not return `route_service_url` in response to the create binding request. By sending `bind_resource` as described above, the platform enables dynamic configuration of a service instance already in the application request path for the route, requiring no change in the platform routing tier.
+- To configure a service instance with behavior specific to an application's
+  routable address, a platform MUST send the address along with the request
+  to create a binding using `"bind_resource":{"route":"some-address.com"}`.
 
-The `requires` field in the [Catalog](#catalog-management) endpoint enables a platform marketplace to validate requests to create bindings. A platform MAY reject requests to create bindings when a broker has declared `"requires":["route_forwarding"]` for a service in the catalog endpoint.
+- Upon receiving an incoming application request, the platform needs to know
+  where to send this request to. To facilitate this, the broker MUST return
+  a `route_service_url` in the response for a request to create a binding.
+  However, if out-of-band configuration was performed then brokers MUST NOT
+  return return a `route_service_url` parameter.
```

Comment:
I feel like some clarity has been lost in the new version. Suggestion:

"Some platforms support proxying of application requests to service instances. If a service is deployed in a configuration to support this behavior, the broker MUST return a `route_service_url` in the response for a request to create a binding, so that the platform knows where to send the application request. If the service is deployed such that the network configuration to proxy application requests through instances of the service is managed out-of-band, the broker MUST NOT return `route_service_url` in the response."